### ISSUE TITLE
[Doc]: Document Completion Program

### ISSUE THREAD

**Issue by freemankevin** [eeshugerman/postgres-backup-s3#33]
Is there a blog demo tutorial? I still get various errors when I get started for the first time. Thank you.

**Comment by eeshugerman** [comment]
Nope, the README is the only documentation. If you have specific complaints about the README or you are running into specific errors, you are welcome to open an issue describing the problem.
